jupyterhub/app.py

```diff
@@ -1163,8 +1162,7 @@ def init_oauth(self):
         self.oauth_provider = make_provider(
             self.session_factory,
             url_prefix=url_path_join(base_url, 'api/oauth2'),
-            login_url=url_path_join(base_url, 'login')
-,
+            login_url=url_path_join(base_url, 'login'),
```

Comment:
I don't know that we technically need a comma here as `make_provider` only takes 3 arguments.